.flowconfig

```diff
@@ -14,7 +14,7 @@
 [libs]
 
 [options]
-module.name_mapper='gymnast' -> '<PROJECT_ROOT>/src'
+module.name_mapper='gymnast' -> '<PROJECT_ROOT>/dist/gymnast'
```

Comment:
This should hopefully give us a better confidence that flow types are working correctly